cns/restserver/ipam_test.go

```diff
@@ -1625,7 +1625,7 @@ func TestIPAMGetK8sSWIFTv2IPSuccess(t *testing.T) {
 	// Asserting that SWIFT v2 IP is returned
 	assert.Equal(t, SWIFTv2IP, podIPInfo[2].PodIPConfig.IPAddress)
 	assert.Equal(t, SWIFTv2MAC, podIPInfo[2].MacAddress)
-	assert.Equal(t, cns.DelegatedVMNIC, podIPInfo[2].NICType)
+	assert.Equal(t, cns.NodeNetworkInterfaceFrontendNIC, podIPInfo[2].NICType)
```

Comment:
can we add a e2e test in ipam_test for our scenario as well? if windows path cannot be covered, we should test for the other 2 nic types data, to verify that the podIpInfo.Nictype returned is accelnet & backendnic for those ncs